src/main/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepository.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.hmc.repository;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.hmc.data.HearingEntity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Transactional(propagation = Propagation.REQUIRES_NEW)
+@Repository
+public interface HearingResponseRepository extends CrudRepository<HearingEntity, Long> {
+
+    @Query("select hre.partiesNotifiedDateTime from HearingResponseEntity hre WHERE hre.hearing.id = :hearingId "
+            + "AND hre.partiesNotifiedDateTime is NOT NULL")
+    List<LocalDateTime> getHearingResponses(Long hearingId);
```

Comment:
Writing the query is starting to go into the next ticket - not a problem, good of you to start, but since it's here will mention that it would need to return the complete entity rather than just the notified timestamp to be able to populate the response ultimately required. (Please ignore if this was just put as a placeholder to return _something_ for the purpose of this ticket, can deal with it in the query/response ticket)
```suggestion
    @Query("select hre from HearingResponseEntity hre WHERE hre.hearing.id = :hearingId "
            + "AND hre.partiesNotifiedDateTime is NOT NULL")
    List<HearingResponseEntity> getHearingResponses(Long hearingId);
```

Reply:
Thanks. Yes, just a placeholder for now.